app/models/user.rb

```diff
@@ -33,6 +29,6 @@ def parsed_auth_first_name(auth_data)
   end
 
   def parsed_auth_last_name(auth_data)
-    auth_data["info"]["name"].split(" ")[1]
+    auth_data["info"]["name"].split(' ')[1]      
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.